src/shogun/base/SGObject.cpp

```diff
@@ -80,6 +81,11 @@ namespace shogun
 			map.at(tag).set_value(value);
 		}
 
+		AnyParameter& at(const BaseTag& tag)
```

Comment:
might also need a const version of this?

Reply:
sure, might be useful & for completion sake